### DISCUSSION TITLE
Bluetooth

### DISCUSSION THREAD

**Discussion by MultimediaLucario** [valnoxy/t480-oc#100] [General]
I just bought a Lenovo ThinkPad T480 and installed 13.4 Ventura on it but Bluetooth doesn't seem to work. Is there anything you can recommend that would help?

**Comment by iamtwan** [comment]
Were you able to get this working?

**Reply by gustavopicheki** [reply]
Same problem here! Please let me know if you guys get bluetooth working in Ventura!
